src/TorchSharp/NN/Sequential.cs

```diff
@@ -36,6 +37,9 @@ public void Add(string name, torch.nn.Module submodule)
                 _names.Add(name);
             }
 
+            public void Add(torch.nn.Module module)
+                => Add(new Random().NextDouble().ToString()[2..], module);
```

Comment:
See my comment on using random numbers in the module names later on in the file.